<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deposit</class>
 <widget class="QDialog" name="Deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>639</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>487</width>
    <height>639</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>487</width>
    <height>639</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DepositCalculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(182, 182, 182);</string>
  </property>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Срок размещения, месяцев</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sum_tax_value">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>560</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_sum_all_value">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>520</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_rate_deposit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Процентная ставка, %</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_time">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;Times New Roman&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="maximum">
    <number>1200</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>430</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sum_rate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Начисленные проценты</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sum_deposit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Сумма вклада, рублей</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_rate_deposit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;Times New Roman&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="maximum">
    <double>500.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_sum_rate_value">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>480</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calc">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>430</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sum_tax">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Сумма налога*</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_type_capital">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;Times New Roman&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <item>
    <property name="text">
     <string>Нет</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ежемесячно</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ежеквартально</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в полгода</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ежегодно</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_capital">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Периодичность капитализации</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_sum_deposit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;Times New Roman&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="maximum">
    <double>999999999999.989990234375000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_sum_all">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Сумма вклада к концу срока</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_rate_tax">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Налоговая ставка, %</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_rate_tax">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>140</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;Times New Roman&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="maximum">
    <double>500.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_rate_CB">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Ключевая ставка ЦБ, %</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_rate_CB">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;Times New Roman&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="maximum">
    <double>500.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_type_add">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;Times New Roman&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <item>
    <property name="text">
     <string>Нет</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ежемесячно</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ежеквартально</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в полгода</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ежегодно</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_add">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Возможность пополнения</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_type_get">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>340</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;Times New Roman&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <item>
    <property name="text">
     <string>Нет</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ежемесячно</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ежеквартально</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в полгода</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ежегодно</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_get">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Возможность частичного снятия</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_note">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>600</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 10pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>*налог на вклады за 2021 и 2022 год отменен</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_sum_add">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>300</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;Times New Roman&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="maximum">
    <double>1000000000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_add_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Сумма пополнения</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_get_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Сумма частичного снятия</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_sum_get">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;Times New Roman&quot;;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="maximum">
    <double>1000000000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
